core/src/main/kotlin/org/evomaster/core/search/service/Statistics.kt

```diff
@@ -35,7 +35,7 @@ class Statistics : SearchListener {
     @Inject(optional = true)
     private var sampler: Sampler<*>? = null
 
-    @Inject(optional = true)
+    //@Inject(optional = true)
     private var remoteController: RemoteController? = null
```

Comment:
we had a discussion about this at a certain point, but I don't remember :(  can u please remind me what it was? (maybe I was supposed to do some fixes, and then I forgot?)

Reply:
hi Andrea. the problem is that when exporting statistics info with `Statistics`, `remoteController` is always Instantiated. But `remoteController` may not exist, in this case, a null pointer exception will be thrown by `val unitsInfo = remoteController?.getSutInfo()?.unitsInfoDto`